src/app/pages/HomePage/index.tsx

```diff
@@ -68,10 +70,11 @@ const FooterStyled = styled(Box)(() => ({
 }))
 
 export const HomePage: FC = () => {
+  const apiStatusQuery = useGet()
```

Comment:
`useGet` name shall be improved after https://github.com/oasisprotocol/oasis-indexer/pull/302 :D

Reply:
cool, thx!

Reply:
Already fixed here: https://github.com/oasisprotocol/explorer/pull/102/commits/46806b7da0a195f634d80f9065a1f2acf2f55997 

Just review and merge #102.